deep_translator/constants.py

```diff
@@ -108,24 +109,146 @@
     'sv': 'swedish',
     'tg': 'tajik',
     'ta': 'tamil',
+    'tt': 'tatar',
     'te': 'telugu',
     'th': 'thai',
     'tr': 'turkish',
+    'tk': 'turkmen',
     'uk': 'ukrainian',
     'ur': 'urdu',
+    'ug': 'uyghur',
     'uz': 'uzbek',
     'vi': 'vietnamese',
     'cy': 'welsh',
     'xh': 'xhosa',
     'yi': 'yiddish',
     'yo': 'yoruba',
     'zu': 'zulu',
-    'fil': 'Filipino',
-    'he': 'Hebrew'
```

Comment:
I would like to mention as per the translator database, ```Hebrew and Filpino``` are uniquely abbreviated as defined:
```
'tl': 'filipino'
'iw': 'hebrew'
```

Now I need to remove the above languages because they were redundant earlier both languages were already described above in the dictionary itself and moreover, as per the translator database we don't have any abbreviations like ```fil``` and ```he```
I would like to pin it that Filipino and filipino are not two different languages same is true for hebrew and Hebrew.

************************************
Yes, the languages that I added is supported by the translator website, you can check out for uyghur 
https://translate.google.com/m?sl=ug&tl=pt&hl=en

I have tallied each and every abbreviation with the languages so you need not worry.

Reply:
ah alright, yes this is a good catch, thanks